vimcat

```diff
@@ -6,13 +6,14 @@
 : if 0
     link="$0"
 
-    while true; do
+    while [ -h "$link" ]; do
         ls0="`ls -l \"$link\"`"
         new_link="`expr \"$ls0\" : '.* -> \(.*\)$'`"
-
-        [ -z "$new_link" ] && break
-
-        link="$new_link"
+        if expr "$new_link" : '/.*' > /dev/null; then
+            link="$new_link"
+        else
+            link="$(dirname $link)/$new_link"
```

Comment:
No. I think that should be mentioned somewhere more general. Somewhere in the docs. There are already comments in `inc/prologue.sh` about being POSIX after sourcing it.

 @rkitover and I had some longer discussions and clarifications about such topics in other issues recently. Mostly #149 and #159